code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -665,16 +665,16 @@
 /obj/item/weapon/reagent_containers/food/snacks/cookie/holiday/New()
 	..()
 
-	var/NM = time2text(world.realtime,"MM")
+	var/NM = time2text(world.realtime,"Month")
 	var/cookiecutter = null
 
 	if(!cookiecutter)
```

Comment:
There's no need for this if-statement at all. `cookiecutter `will always be null here. You should get rid of the if and make the cookiecutter definition just `var/cookiecutter`.